src/gdash/scripts/_contains/_contains.gml

```diff
@@ -15,9 +15,10 @@ _contains("hello", "ello");
 // => true
 */
 
-var collection = argument[0];
-var target = argument[1]
-var fromIndex = 0;
+var 

```

Comment:
I'm happy to pull this down and run the tests. Once I get the coverage to a place that I feel is good enough, I'll pull this down and see how it's doing. I can also make the style changes I'd like to see.

This is really great stuff and I'd like to get it in to make sure that gdash is the best it can be, so I'm happy to work on getting this in.